local_units/management/commands/import_local_units_health.py

```diff
@@ -0,0 +1,47 @@
+from django.db import transaction
+import csv
+import pytz
+from dateutil import parser as date_parser
+from django.core.management.base import BaseCommand
+from django.contrib.gis.geos import Point
+
+
+from api.models import Country
+from ...models import LocalUnit, LocalUnitType
+
+
+class Command(BaseCommand):
+    help = "Import LocalUnits data from CSV"
+    missing_args_message = "Filename is missing. Filename / path to CSV file required. Required headers in CSV: distr_code, GOadm2code, ADM2"
+
+    def add_arguments(self, parser):
+        parser.add_argument('filename', nargs='+', type=str)
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        filename = options['filename'][0]
+        with open(filename) as csvfile:
+            reader = csv.DictReader(csvfile)
+            for i, row in enumerate(reader):
+                # Without positions we can't use the row:
+                if not row['LONGITUDE'] or not row['LATITUDE']:
+                    continue
```

Comment:
Let's add a warning here
```suggestion
                    self.stdout.write(
                        self.style.WARNING(f'Skipping row {i}: Empty locations point')
                    )
                    continue
```